### DISCUSSION TITLE
How to import extension types using starterkit ? (ie Heading Level type)

### DISCUSSION THREAD

**Discussion by johann-taberlet** [ueberdosis/tiptap#3584] [General Questions]
Hi,
everything is in the title. I use typescript and I struggle to import the Level type of the heading extension.

```ts
const toggleHeading = (lvl:number) => {
    editor.chain().focus().toggleHeading({ level: lvl }).run();
};
```

Without this type I can't cast my lvl number to make typescript happy.

Any ideas ?

**Comment by gethari** [comment]
The `Level` is exported only from `extension-heading` package at [Line 3](https://github.com/ueberdosis/tiptap/blob/62d42275c529aa3c819e85133b0ad0dd22cac934/packages/extension-heading/src/heading.ts#LL3C15-L3C15). I don't think you can import it from the `starter-kit`
